tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
```

Comment:
glad to see tox setup here. Some of the flake8 tests fail but not a big deal as this is all bonus adds